server/api/services/service.ts

```diff
@@ -0,0 +1,43 @@
+import * as mongoose from "mongoose";
+import * as fetch from "node-fetch";
+import { URLSearchParams } from "url";
+import { UserModel } from "../models/user";
+
+const User = mongoose.model("User", UserModel);
+
+export class ServiceService {
```

Comment:
`ServiceService` name sounds ambiguous, maybe something like `ServiceProvidersService`?

Reply:
I agree. Going for ServiceProviderService.